### PR TITLE
Course Model GraphQL Migration

### PR DESCRIPTION
**Pull request by dhuangg**
Migrated - Registration Landing and Registration Course

### CODE REVIEW

File: src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ import Moment from "react-moment";
 
 import {courseDateFormat} from "utils";
 
+export const GET_INSTRUCTOR_ENROLLMENTS = `
+  query InstructorEnrollments($instructorId) { 
+    courses(filter: { instructor.user.id: $instructorId){
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js:20]: Is this formatted properly?

File: src/components/FeatureViews/Accounts/TabComponents/StudentCourseViewer.js [hunk 2]
```diff
@@ -172,26 +143,30 @@ const StudentCourseViewer = ({ studentID, current = true }) => {
...
 													}
 												/>
 												{` - `}
 												<Moment
 													format="h:mm a"
 													date={
-														course.schedule.end_date + course.schedule.end_time
+														enrollment.course.endDate
+														+ "T" +
+														enrollment.course.endTime
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/StudentCourseViewer.js:157]: is there no better way to format this lol

File: src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js (hunk 1 above)
Comment (dhuangg) [reply src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js:20]: this is still a work in progress haha. not done with this one, but also not sure what the syntax should be here

File: src/components/FeatureViews/Accounts/TabComponents/StudentCourseViewer.js (hunk 2 above)
Comment (dhuangg) [reply src/components/FeatureViews/Accounts/TabComponents/StudentCourseViewer.js:157]: you gotta make it a datetime string somehow lol. you could make it a template literal i guess haha

File: src/utils.js [hunk 3]
```diff
@@ -372,3 +372,9 @@ export const useGoToRoute = () => {
 export const uniques = (array) => array.filter(
     (element, index, filteredArray) => filteredArray.indexOf(element) === index
 );
+
+/***
+ * Returns full name given a user object
+ * @param {Object} user
+ */
+export const fullName = (user) => `${user.firstName} ${user.lastName}`;
```
Comment (NeilAllavarpu) [inline src/utils.js:380]: Can use destructuring here.

File: src/components/FeatureViews/Registration/PaymentReceipt.js [hunk 4]
```diff
@@ -1,152 +1,109 @@
...
+	}
+`;
+
+const PaymentReceipt = ({paymentID}) => {
 	const history = useHistory();
+	const location = useLocation();
+	const params = useParams();
+
+	const {data, loading, error} = useQuery(GET_PAYMENT,
+		{variables: {paymentId: params.paymentID || paymentID}}
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Registration/PaymentReceipt.js:68]: We should probably choose one or the other to use (unless we need both use cases?)

Comment (dhuangg) [reply src/components/FeatureViews/Registration/PaymentReceipt.js:68]: yeah, we need both use cases - one for when you go onto a payment page, the other for when you're completing a registration flow.

**Pull request merged**